Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -195,7 +201,9 @@ private void CreateRange()
             else
             {
                 this.lower = limits[0];
-                this.upper = limits[1];
+                this.upper = limits[1] > Int32.MaxValue
+                    ? limits[1]
+                    : limits[1] + 1;

```

Comment:
@jryanmoseley, thank you, it does :+1: 

This is one of the rare cases where I'd think that a code comment might be in place.

One of the facts that mislead me was that it wasn't clear to me why `Int32.MaxValue` was particularly interesting, since a request could also be for an `Int64` value. However, the fact that `Random.Next(int, int)` operates on `Int32` instead of `Int64` explains that part.